src/palo_alto_firewall_analyzer/scripts/pan_analyzer.py

```diff
@@ -152,6 +202,7 @@ def main():
 
     parser.add_argument("--debug", help="Write all debug output to pan_validator_debug_YYMMDD_HHMMSS.log", action='store_true')
     parser.add_argument("--limit", help="Limit processing to the first N rules (useful for debugging)", type=int)
+    parser.add_argument("--output", help="Type File Output (text, json), default = text", type=str)
```

Comment:
This looks like a typo. Instead of:
`default = text"`, it should probably be `default="text"`.

It also looks like I really need to add some CI/CD jobs for testing, because a unit test would have caught that one.

Reply:
fixed

Reply:
Sorry, I was unclear. the argument for `default` actually lets you set a default value, so you don't need to check if it's None later.

So instead of:
```python
import argparse
parser = argparse.ArgumentParser(description="demo")
parser.add_argument("--arg", help='default is "abc"', choices=["abc", "def"])
parsed_args = parser.parse_args()

if parsed_args.arg is None:
    myarg = "abc"
else:
    myarg = parsed_args.arg

print(myarg)
```
Use:
```python
import argparse
parser = argparse.ArgumentParser(description="demo")
parser.add_argument("--arg", help='default is "abc"', choices=["abc", "def"], default="abc")
parsed_args = parser.parse_args()

myarg = parsed_args.arg
print(myarg)
```


parser.add_argument("--output-format", help="File output format", default="text", type=str, choices=["text", "json"])